extra/migrate_settings.rb

```diff
@@ -91,7 +91,7 @@ def create_migration_dirs
     end
 
     def underscore(src)
-      src = src.gsub(/::/, '/')
+      src = src.gsub("::", '/')
```

Comment:
would `'::'` also work? (I would have expected :police_officer: to yell at you)

Reply:
In this repo that cop is disabled. I'm not sure why RuboCop would prefer `"`.

Reply:
Ah, very well, then something for the future.